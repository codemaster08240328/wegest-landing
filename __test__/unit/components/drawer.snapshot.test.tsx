import renderer from 'react-test-renderer'
import Drawer from '@/components/layout/drawer'

describe('<Drawer> snapshot', () => {
  it('renders without icon', () => {
    expect(renderer.create(<Drawer />).toJSON()).toMatchSnapshot()
  })

  it('renders with icon', () => {
    expect(renderer.create(<Drawer icon='email' />).toJSON()).toMatchSnapshot()
  })
})
