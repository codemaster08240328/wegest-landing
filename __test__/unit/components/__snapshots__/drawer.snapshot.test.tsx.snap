// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Drawer> snapshot renders with icon 1`] = `
Array [
  <button
    className="MuiButtonBase-root MuiIconButton-root Component-root-1 Component-root-3"
    data-testid="el-icon-button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiIconButton-label"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
        data-testid="el-icon"
      >
        email
      </span>
    </span>
    <span
      className="MuiTouchRipple-root"
    />
  </button>,
  <div
    className="PrivateSwipeArea-root-4 PrivateSwipeArea-anchorLeft-5"
    style={
      Object {
        "width": 20,
      }
    }
  />,
]
`;

exports[`<Drawer> snapshot renders without icon 1`] = `
<button
  className="MuiButtonBase-root MuiIconButton-root Component-root-1 Component-root-2"
  data-testid="el-icon-button"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiIconButton-label"
  >
    <span
      aria-hidden={true}
      className="material-icons MuiIcon-root"
      data-testid="el-icon"
    >
      menu
    </span>
  </span>
</button>
`;
